client/blocks/plugins-scheduled-updates-multisite/schedule-form-sites.tsx

```diff
@@ -55,17 +55,25 @@ export const ScheduleFormSites = ( props: Props ) => {
 						{ error }
 					</Text>
 				) }
-				<SearchControl
-					id="sites"
-					value={ searchTerm }
-					onChange={ ( s ) => setSearchTerm( s.trim() ) }
-					placeholder={ translate( 'Search sites' ) }
-				/>
+				{ !! sites.length && (
+					<SearchControl
+						id="sites"
+						value={ searchTerm }
+						onChange={ ( s ) => setSearchTerm( s.trim() ) }
+						placeholder={ translate( 'Search sites' ) }
+					/>
+				) }
 				{ ! sites.length && (
-					<Text className="info-msg">
-						You can only select sites with Creator plan. Please upgrade your site to enable this
-						feature.
-					</Text>
+					<p className="placeholder-info">
+						{ translate(
+							'You can only select sites with Creator plan.{{br/}}Please upgrade your site to enable this feature.',
```

Comment:
```suggestion
							'To select a site, please ensure it has a Creator plan or higher.{{br/}}Upgrade your site to proceed.',
```
Is this an opportunity to make the plan name dynamic so it doesn't have to be updated manually the next time we change plan names?
Is there an easy way for customers to upgrade a site from here?

Reply:
> Is this an opportunity to make the plan name dynamic

Absolutely, that's a good point. I'll update it. Thanks for the copy suggestion; it's more appropriate for this case.

> Is there an easy way for customers to upgrade a site from here?

I am not sure if this is an option since there can be different flows—having one or multiple simple sites and creating a new site. I don't think it's worth spending time handling those cases.